src/tokenizer/tokenizer.py

```diff
@@ -721,9 +727,10 @@ def gen_from_string(txt, replace_composite_glyphs=True, replace_html_escapes=Fal
             yield ""
         for w in span.split():
             yield w
+        if onesentperline:
+            yield " "
```

Comment:
1) Are you sure this is needed? And 2) should it be "" instead of " "?

Reply:
Tested, not needed. Deleted!